portfolio/src/main/webapp/index.html

```diff
@@ -5,17 +5,57 @@
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <style>
+    a    {
+        color: black;
+        text-shadow: 2px 2px 5px turquoise, 1px 1px 3px purple, 1px 2px 3px snow;
+        text-decoration: underline;
+    }
+
+    body {
+        background: url(images/better.jpg) no-repeat center center fixed;
+    }
+
+    html {
+        scroll-behavior: smooth;
+    }
+
+    img  {
+        border-radius: 30px;
+    }
+
+    img:hover {
+        box-shadow: 0 0 2px 1px white;
+    }
+    </style>
   </head>
   <body>
     <div id="content">
-      <h1>Kyle Smith's Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>I am a rising sophomore at MIT who is studying Mathematics & Computer Science.</p>
-      <p>I am an intern at Google</p>
-      <img src="/images/mit.jpg" width="300" height="177" alt="This is my university">
-      <p>Click here to get a random fact about me:</p>
-      <button onclick="addRandomFact()">Fact!</button>
-      <div id="fact-container"></div>
+      <div id="bold"><h1>Kyle Smith's Portfolio</h1></div>
+
+      <p><a href="https://github.com/kyleasmith1">GitHub</a>&nbsp;&nbsp;
+      <a href="about.html">About Me</a>&nbsp;&nbsp;
+      <a href="https://linkedin.com/in/k-smith23">LinkedIn</a></p>
+
+      <div id="bold"><h3>Click Here To See Some Cool Places I've Been To:</h3></div>
```

Comment:
I believe you can edit the h3 tag in CSS instead of creating a new div tag. This will make it so that your other h3 will have the same properties without having to make a bold div.

Reply:
Correct. Also, as a general rule, it is worth  targeting things as generally as possible. So beyond just putting the id on the h3 directly, you can also just target all h3s (as Andrew suggested), or if there is some category of h3s which need to be bolded (say most h3s should be left along but this is of a certain kind) then adding and targeting a class in CSS makes it reusable (for example of you want some h3s to be left alone, but others not to be)